src/components/timeline/LocalTrack.js

```diff
@@ -71,9 +70,7 @@ class LocalTrackComponent extends PureComponent<Props> {
     } else if (event.button === 2) {
       // This is needed to allow the context menu to know what was right clicked without
       // actually changing the current selection.
-      if (trackReference) {
-        changeRightClickedTrack(trackReference);
-      }
+      changeRightClickedTrack({ type: 'local', pid, trackIndex });
```

Comment:
I opted to remove the TrackReference from the props, so I wouldn't have to worry about props strict equality with recreating the object.